x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality_details/degraded_field_flyout/possible_mitigations/field_limit/field_mapping_limit.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiAccordion,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import {
+  degradedFieldCurrentFieldLimitColumnName,
+  fieldLimitMitigationConsiderationText,
+  fieldLimitMitigationConsiderationText1,
+  fieldLimitMitigationConsiderationText2,
+  fieldLimitMitigationConsiderationText3,
+  fieldLimitMitigationConsiderationText4,
+  fieldLimitMitigationDescriptionText,
+  increaseFieldMappingLimitTitle,
+} from '../../../../../../common/translations';
+import { useDegradedFields } from '../../../../../hooks';
+import { IncreaseFieldMappingLimit } from './increase_field_mapping_limit';
+import { FieldLimitDocLink } from './field_limit_documentation_link';
+import { MessageCallout } from './message_callout';
+
+export function FieldMappingLimit({ isIntegration }: { isIntegration: boolean }) {
+  const accordionId = useGeneratedHtmlId({
+    prefix: increaseFieldMappingLimitTitle,
+  });
+
+  const { degradedFieldAnalysis } = useDegradedFields();
+
+  const accordionTitle = (
+    <EuiTitle size="xxs">
+      <h6>{increaseFieldMappingLimitTitle}</h6>
+    </EuiTitle>
+  );
+
+  return (
+    <EuiPanel hasBorder grow={false}>
+      <EuiAccordion
+        id={accordionId}
+        buttonContent={accordionTitle}
+        initialIsOpen={true}
+        data-test-subj="datasetQualityDetailsDegradedFieldFlyoutFieldLimitMitigationAccordion"
+        paddingSize="s"
+      >
+        <EuiText size="xs">
+          <p>{fieldLimitMitigationDescriptionText}</p>
+        </EuiText>
```

Comment:
```suggestion
        <EuiText component="p" size="xs">
          {fieldLimitMitigationDescriptionText}
        </EuiText>
```
Check out also all the other places where this suggestion may apply, is a nice addition the EUI team added to avoid polluting the DOM with too many wrappers.